docker-compose.yml

```diff
@@ -1,5 +1,6 @@
 services:
   postgres:
+    platform: linux/amd64
```

Comment:
This image doesn't currently support ARM64 - https://github.com/postgis/docker-postgis/issues/216

Though the container seemed to be spinning up without issue when using `DOCKER_DEFAULT_PLATFORM=linux/arm64`, it was leading to Hasura migrations hanging and ultimately failing.

By specifying `linux/amd64` here, we can force emulation and a healthy build.